src/styles/index.scss

```diff
@@ -1,3 +1,89 @@
+@import './utils/variables';
+
 body {
   margin: 0;
+  height: 100vh;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+.stopwatch {
+  position: relative;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+
+  width: $stopwatch-size;
+  height: $stopwatch-size;
+  border: $border-width dotted $color-border;
+  border-radius: 50%;
+
+  &__center {
+    position: absolute;
+    width: $center-diameter;
+    height: $center-diameter;
+    background-color: $color-center;
+    border-radius: 50%;
+    z-index: 1;
+  }
+
+  &__hand {
+    position: absolute;
+
+    &--minutes {
+      width: $minute-hand-width;
+      height: $minute-hand-length;
+      top: 20vmin;
```

Comment:
The 'top' property value '20vmin' seems unusual for positioning the minute hand. Usually, the hands of a stopwatch are centered. Verify if this value positions the hand as intended and consider using a more conventional unit like 'px', 'em', or 'rem' if this is not the case.